x-pack/platform/plugins/shared/fleet/server/routes/agent/index.ts

```diff
@@ -1001,4 +1005,42 @@ export const registerAPIRoutes = (router: FleetAuthzRouter, config: FleetConfigT
         changeAgentPrivilegeLevelHandler
       );
   }
+  if (experimentalFeatures.enableAgentPrivilegeLevelChange) {
+    router.versioned
+      .post({
+        path: AGENT_API_ROUTES.BULK_PRIVILEGE_LEVEL_CHANGE_PATTERN,
+        security: {
+          authz: {
+            requiredPrivileges: [FLEET_API_PRIVILEGES.AGENTS.ALL],
+          },
+        },
+        summary: `Bulk change agent privilege level`,
+        description: `Change multiple agents' privilege level to unprivileged.`,
+        options: {
+          tags: ['oas-tag:Elastic Agents'],
+          availability: {
+            since: '9.2.0',
```

Comment:
we should modify this to 9.3, probably the feature won't be ready until 9.2 FF (in a few days)